### PR TITLE
More configurable dumping

### PR DESCRIPTION
**Pull request by NcUltimate**
🍏 📗 💚 

Extends `pgmgr` to allow for more highly configurable dumping options as specified in https://www.postgresql.org/docs/11/app-pgdump.html :
- `exclude-schemas`: The `-N` option
- `seed-tables`: The `-t` option, only used with `--data-only`
- `dump-file`: Same as the current `dump-file` but promoted to within the dump config instead of being a top-level option.
- `compress`: (`'t'`/`'f'`) whether to use compression on the dump file (default `'t'`)
- `include-triggers`: (`'t'`/`'f'`) whether to fire triggers when dumping the db (default `'f'`)
- `include-privileges`: (`'t'`/`'f'`) whether to include GRANT/REVOKE acls in the dump file (default `'f'`)

### CODE REVIEW

**Comment by rnubel** [comment]
One thought -- pgmgr currently allows all of its configuration properties to be set by env vars, CLI flags, *or* the JSON. I'm okay with not keeping that up for these nested params, unless someone has a use case for it, but we might want to think about adding deprecation warnings to those params at the CLI level.

File: pgmgr/dump_config.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		config.ExcludeData = ctx.StringSlice("exclude-table-data")
+	}
+	if sliceValuesGiven(ctx, "include-schemas") {
+		config.IncludeSchemas = ctx.StringSlice("include-schemas")
+	}
+	if sliceValuesGiven(ctx, "include-tables") {
+		config.IncludeTables = ctx.StringSlice("include-tables")
+	}
+	if sliceValuesGiven(ctx, "seed-tables") {
+		deprecatedDumpFieldWarning("seed-tables", "include-tables", "command line arg")
```
Comment (rnubel) [inline pgmgr/dump_config.go]: Actually, `seed-tables` and `include-tables` are not equivalent. `seed-tables` would line up with an `include-table-data` flag, which I don't see as part of this PR. That is, include/exclude tables refers to tables which are having their structures dumped; the old pgmgr behavior was to dump everything and then just the data for `seed-tables`.

In other words, if I wanted to configure pgmgr to dump my database but exclude tables X Y and Z entirely and only seed tables A B and C, is that possible?

Comment (NcUltimate) [reply pgmgr/dump_config.go]: Good catch, this change would make it possible to exclude X Y and Z but not possible to fetch _only_ the data for A B and C. Unfortunately there is no flag for grab _only_ the data from specific tables -- we would still have to use `--data-only` which affects the whole dump.

I'll reintroduce (and alias) `seed-tables` as `data-only-tables` and rename `exclude-table-data` to `schema-only-tables`. to support this change I would need to also add the functionality back in that dumps twice: once with `--data-only` and once without it

Comment (NcUltimate) [reply pgmgr/dump_config.go]: We discussed this further -- we'll try to boil all of these options down to just `exclude-schemas` and `seed-tables`. A `--schema-only` dump will be performed with all `exclude-schemas` omitted, and a `--data-only` dump will be performed with only `seed-tables` tables included (as the current version already does)

**Pull request merged**